tests/test_sbr_calculator.py

```diff
@@ -0,0 +1,353 @@
+from src.smart_building_rating_calculator.calculate_sbr_score import (
+    get_sbr_scores,
+    sbr_score,
+)
+from src.smart_building_rating_calculator.flex_archetype import FlexArchetype
+from src.smart_building_rating_calculator.inputs import (
+    BatterySize,
+    EVChargerPower,
+    HeatingSource,
+    HotWaterSource,
+    SolarInverterSize,
+    UserInputs,
+)
+from src.smart_building_rating_calculator.intermediate_scoring import (
+    calc_electrification_score,
+    calc_ics_score,
+    calc_smart_meter_score,
+)
+
+
+class TestScoreCalculators:
+    """Test the scoring calculations using the results in the SBR excel sheet created by ESC"""
+
+    def test_smartest_home(self):
+        user_inputs = UserInputs(
+            smart_meter=True,
+            smart_ev_charger=True,
+            charger_power=EVChargerPower.CHARGER_7KW,
+            smart_v2g_enabled=True,
+            home_battery=True,
+            battery_size=BatterySize.LARGE,
+            solar_pv=True,
+            pv_inverter_size=SolarInverterSize.LT_4KW,
+            electric_heating=True,
+            heating_source=HeatingSource.HEAT_PUMP,
+            hot_water_source=HotWaterSource.HEAT_BATTERY_OR_ELEC_HOT_WATER_TANK,
+            secondary_heating=True,
+            secondary_hot_water=True,
+            integrated_control_sys=True,
+        )
+        elec_scores = calc_electrification_score(user_inputs)
+        smart_meter_score = calc_smart_meter_score(user_inputs)
+        ics_score = calc_ics_score(user_inputs)
+        elec_scores.insert(0, smart_meter_score)
+        elec_scores.insert(len(elec_scores), ics_score)
+
+        expected_scores = [1, 3, 3, 1, 3, -1, 4, 2, 1, 1, 1.5]
```

Comment:
The purpose of unit tests is not just to test correct input = correct output, but also to add syntax to explain to people what the tests are actually checking.

Writing expected scores like these are not giving developers any syntax on what these numbers mean or why we should be expecting this results.

Reply:
Let me know if better now please. Thanks!

Reply:
Note that I am taking the expected scores from ESC's spreadsheet - I don't know what the numbers mean or why we should be expecting this results, other than I expect these values based on the calculations ESC have provided!